src/admin.ts

```diff
@@ -20,4 +20,34 @@ export class Admin {
     );
     return response.data.token as string;
   }
+  async getAvailableFunctions({ customerId, namespace }: Runnable) {
+    const response = await axios.get(
+      `${this.baseUrl}/functions/${customerId}/${namespace}`
+    );
+    return response.data;
+  }
+  async getFunctionResults({
+    environment,
+    customerId,
```

Comment:
@ospencer are all of these supposed to be un-typed?

Reply:
good point, the results should be typed `as object` yeah?

Reply:
These all get typed via the `: VersionedRunnable` a couple of lines down—no need to add `as object` here 🙂

Reply:
@ospencer I realize `getAvailableFunctions` isn't typed by `VersionedRunnable` even though it's an object. So I'm assuming I would specify an object there.

Reply:
Update: they should be typed. Going to create three different interfaces and then save the response.data as the name of the interface :)